Framework/CurveFitting/test/FuncMinimizers/FABADAMinimizerTest.h

```diff
@@ -190,23 +190,23 @@ class FABADAMinimizerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(PDF->getNumberHistograms(), nParams + 1);
     TS_ASSERT_EQUALS(PDF->x(0).size(), 21);
     TS_ASSERT_EQUALS(PDF->y(0).size(), 20);
-    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.01);
-    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 0.01);
-    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.01);
+    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.3);
+    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 1.0);
+    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.3);
```

Comment:
Bayesian analysis should be deterministic in the limit of a large number of iterations. This is because the acceptance rate is based on the probability distribution function, so after many iterations the statistics will average out even if different random numbers are used. 

If the minimizer was not deterministic then tests could never work, as it would not find a different minima each time the test is ran